### PR TITLE
[console] - fixed issues with task dialog flashing

### PR DESCRIPTION
**Pull request by emilbon99**
# Feature Pull Request Template

## Key Information

- **Linear Issue**: [SY-1446](https://linear.app/synnax/issue/SY-1446/fix-task-flashing)

## Description

Fixes issues with CSS container queries that causes certain Modal components to flash.

## Basic Readiness

- [x] I have performed a self-review of my code.
- [x] I have added relevant tests to cover the changes to CI.
- [x] I have needed QA steps to the [release
      candidate](/.github/PULL_REQUEST_TEMPLATE/rc.md) template that cover these changes.
- [x] I have updated in-code documentation to reflect the changes.
- [x] I have updated user-facing documentation to reflect the changes.

## Backwards Compatibility

The following makes sure that this feature does not break backwards compatability.

### Data Structures

- [ ] Server - I have ensured that previous versions of stored data structures are
      properly migrated to new formats.
- [ ] Console - I have ensured that previous versions of stored data structures are
      properly migrated to new formats.

### API Changes

- [ ] Server - The server API is backwards-compatible
- The following client APIs are backwards-compatible:
  - [ ] C++
  - [ ] TypeScript
  - [ ] Python

### Breaking Changes

If anything in this section is not true, please list all breaking changes.

### CODE REVIEW

File: console/src-tauri/gen/schemas/windows-schema.json
Comment (pjdotson) [inline console/src-tauri/gen/schemas/windows-schema.json:1]: Why did this change?

**Review by pjdotson — APPROVED** [review]
Approve after you respond to my comment

File: console/src-tauri/gen/schemas/windows-schema.json
Comment (emilbon99) [reply console/src-tauri/gen/schemas/windows-schema.json:1]: Idk, but I rolled it back to a formatted version of the one on main.

Comment (emilbon99) [reply console/src-tauri/gen/schemas/windows-schema.json:1]: Idk, but I rolled it back to a formatted version of the one on main.

**Pull request merged**